packages/flutter_test/lib/src/controller.dart

```diff
@@ -423,7 +423,8 @@ abstract class WidgetController {
   /// See [PointerEventRecord].
   Future<List<Duration>> handlePointerEventRecord(List<PointerEventRecord> records);
 
-  /// Called to indicate that time should advance.
+  /// Called to indicate that there should be a new frame and if necessary,
+  /// time should advance.
```

Comment:
Maybe
```suggestion
  /// Called to indicate that there should be a new frame after an optional delay.
```

And add in the doc
```
  /// The frame is pumped after a delay of [duration] if [duration] is not null,
  /// or immediately otherwise.
```